### PR TITLE
Added sensor properties + example to reflect consistency changes

### PR DESCRIPTION
**Pull request by brentru**
fixes for issue https://github.com/adafruit/circuitpython/issues/622#issuecomment-392822289 and
https://github.com/adafruit/Adafruit_CircuitPython_SGP30/issues

### CODE REVIEW

File: adafruit_sgp30.py [hunk 1]
```diff
@@ -75,6 +75,40 @@ def __init__(self, i2c, address=_SGP30_DEFAULT_I2C_ADDR):
             raise RuntimeError('SGP30 Not detected')
         self.iaq_init()
 
+        self._co2eq = None # pylint: disable=invalid-name
+        self._tvoc = None # pylint: disable=invalid-name
+        self._baseline_tvoc = None # pylint: disable=invalid-name
+        self._baseline_co2eq = None # pylint: disable=invalid-name
```
Comment (tannewt) [inline adafruit_sgp30.py]: You don't need these because you always update them in the property getter.

File: adafruit_sgp30.py [hunk 2]
```diff
@@ -75,6 +75,40 @@ def __init__(self, i2c, address=_SGP30_DEFAULT_I2C_ADDR):
...
+
+
+    @property
+    def tvoc(self): # pylint: disable=invalid-name
+        """Total Volatile Organic Compound in parts per billion."""
+        self._tvoc = self.iaq_measure()[1]
+        return self._tvoc
```
Comment (tannewt) [inline adafruit_sgp30.py]: Just `return self.iaq_measure()[1]` and similar below.

File: adafruit_sgp30.py [hunk 3]
```diff
@@ -75,6 +75,40 @@ def __init__(self, i2c, address=_SGP30_DEFAULT_I2C_ADDR):
...
         self.iaq_init()
 
+        self._co2eq = None # pylint: disable=invalid-name
+        self._tvoc = None # pylint: disable=invalid-name
+        self._baseline_tvoc = None # pylint: disable=invalid-name
+        self._baseline_co2eq = None # pylint: disable=invalid-name
+
+
+    @property
+    def tvoc(self): # pylint: disable=invalid-name
```
Comment (tannewt) [inline adafruit_sgp30.py]: These look like valid names to me. Did you copy the lint part from somewhere else? I don't think they are needed.

**Review by tannewt — CHANGES_REQUESTED** [review]
Thanks for adding these! A couple changes before we check it in.

File: adafruit_sgp30.py (hunk 1 above)
Comment (brentru) [reply adafruit_sgp30.py]: Ok!

File: adafruit_sgp30.py (hunk 3 above)
Comment (brentru) [reply adafruit_sgp30.py]: Yeah, I was copying that from the CCS811 library. I'll remove.

File: adafruit_sgp30.py (hunk 2 above)
Comment (brentru) [reply adafruit_sgp30.py]: Changed - thanks (will keep this in mind next time)!

**Review by tannewt — APPROVED** [review]
Awesome! Thanks for adding these

**Pull request merged**